content/zh/docs/tasks/traffic-management/circuit-breaking/index.md

```diff
@@ -261,5 +261,5 @@ test: yes
 
     {{< text bash >}}
     $ kubectl delete deploy httpbin fortio-deploy
-    $ kubectl delete svc httpbin
```

Comment:
would be better change both the en and zh doc to following:
```
kubectl apply -f @samples/httpbin/sample-client/fortio-deploy.yaml@
kubectl apply -f @samples/httpbin/httpbin.yaml@
```

Reply:
OK, this is a good idea.

Reply:
this's a testable doc, please run `make gen` to update the scripts before submit your commit.